tendrl/commons/message.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+from dateutil import parser
+from inspect import getframeinfo
+from inspect import stack
+import json
+from tendrl.commons.utils.time_utils import now
+import uuid
+
+
+class Message(object):
+    # priority[0] is debug....priority[5] is critical
+    # Use priorities and publishers when call create message obj
+    priorities = type("priorities", (),
+                      {"DEBUG": "debug", "INFO": "info", "NOTICE": "notice",
+                       "WARNING": "warning", "ERROR": "error",
+                       "CRITICAL": "critical"})
+    publishers = type("publishers", (),
+                      {"CEPH_INTEGRATION": "ceph-integration",
+                       "GLUSTER_INTEGRATION": "gluster-integration",
+                       "NODE_AGENT": "node-agent",
+                       "COMMONS": "commons",
+                       "ALERTING": "alerting",
+                       "PERFORMANCE_MONITORING": "performance-monitoring"})
```

Comment:
Commons must not refer to any other components. The components creating the object from the Message class should inject their own publisher id. Each component should have it's own hardcoded publisher id.